suite-native/module-accounts/src/components/AccountBalance.tsx

```diff
@@ -66,16 +66,16 @@ export const AccountBalance = ({ accountKey }: AccountBalanceProps) => {
                     <Box style={applyStyle(cryptoIconStyle)}>
                         <CryptoIcon name={account.symbol} />
                     </Box>
-                    <Text color="gray600" variant="hint">
+                    <DiscreetText color="gray600" typography="hint">
                         {CryptoAmountFormatter.format(cryptoAmount, {
                             symbol: account.symbol,
                         })}
-                    </Text>
+                    </DiscreetText>
                 </Box>
                 <Box>
-                    <Text variant="titleLarge" color="gray800">
-                        <FiatAmountFormatter value={selectedPoint.value} />
-                    </Text>
+                    <DiscreetText typography="titleLarge" color="gray800">
+                        {FiatAmountFormatter.format(selectedPoint.value) ?? ''}
```

Comment:
Please remove all this `?? ''` once you will have default value for DiscreetText